.github/workflows/_mac-test-mps.yml

```diff
@@ -6,7 +6,7 @@ on:
       build-environment:
         required: true
         type: string
-        description: Top-level label for what's being built/tested.
+        description: Top-level label for whats being built/tested.
```

Comment:
nit: "what's" was correct grammer

Reply:
Ah my bad, I accidentally remove this when editing the file. Let me put it back